.github/workflows/build.yml

```diff
@@ -11,55 +11,53 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.9', '3.10', '3.11']
+        python-version: ["3.9", "3.10", "3.11"]
         os: [ubuntu-latest, macOS-latest, windows-latest]
 
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v4
 
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
 
-    - name: Install dependencies
-      run: |
-        pip install -r requirements.txt
-        pip install -r requirements_dev.txt
-        pip install coveralls
-        pip install pytest
-        pip install pytest-benchmark
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install -r requirements_dev.txt
+          pip install coveralls
 
-    - name: Run tests
-      run: coverage run -m unittest discover -s tests -t tests
+      - name: Run tests
+        run: coverage run -m unittest discover -s tests -t tests
 
-    - name: Run benchmark
-      run: python3 -m pytest 
-            --benchmark-verbose 
-            --benchmark-columns=mean,stddev,iqr,ops,rounds 
-            tests/benchmarks/benchmark_model.py
-            tests/benchmarks/benchmark_management_api.py
-            tests/benchmarks/benchmark_role_manager.py
+      - name: Run benchmark
+        run: python3 -m pytest
+          --benchmark-verbose
+          --benchmark-columns=mean,stddev,iqr,ops,rounds
+          tests/benchmarks/benchmark_model.py
+          tests/benchmarks/benchmark_management_api.py
+          tests/benchmarks/benchmark_role_manager.py
 
-    - name: Upload coverage data to coveralls.io
-      run: coveralls --service=github
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        COVERALLS_FLAG_NAME: ${{ matrix.os }} - ${{ matrix.python-version }}
-        COVERALLS_PARALLEL: true
+      - name: Upload coverage data to coveralls.io
+        run: coveralls --service=github
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COVERALLS_FLAG_NAME: ${{ matrix.os }} - ${{ matrix.python-version }}
+          COVERALLS_PARALLEL: true
 
   lint:
     name: Run Linters
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
       - name: Super-Linter
-        uses: github/super-linter@v4.9.2
+        uses: github/super-linter@v5.7.2
```

Comment:
plz use https://github.com/super-linter/super-linter

Reply:
yes, that is the one i am looking at. the version definitely exists: https://github.com/super-linter/super-linter/releases/tag/v5.7.2

Reply:
ah! i see, they moved it from `github` to `super-linter`. my bad.